QAchat/py/QAchat.ipynb

```diff
@@ -93,8 +94,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Upload these to a dataset in BrainTrust so your teammates can also easily use and manage this dataset.\n",
-    "dataset = braintrust.init_dataset(PROJECT_NAME, name=\"Basic Evaluation\")\n",
+    "# Upload these to a new unique dataset in BrainTrust so your teammates can also easily use and manage this dataset.\n",
+    "timestamp = datetime.datetime.now().strftime(\"%Y-%m-%d %H:%M:%S\")\n",
+    "dataset = braintrust.init_dataset(PROJECT_NAME, name=\"Basic Evaluation\"+timestamp)\n",
     "\n",
```

Comment:
Is the purpose of this to avoid inserting duplicate rows? If so, I would suggest one of 2 different things:

- Either specify an `id` in the `insert` statements, and then they'll be automatically overwritten (even an index is fine for this)
- Skip if `len(dataset) > 0`